src/Fixer/FunctionNotation/ReturnTypeDeclarationFixer.php

```diff
@@ -53,8 +103,31 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
     /**
      * {@inheritdoc}
      */
-    protected function getDescription()
+    public function getDefinition()
     {
-        return 'There should be no space before colon and one space after it in return type declaration.';
+        $versionSpecification = new VersionSpecification(70000);
+
+        return new FixerDefinition(
+            'There should be one or no space before colon, and one space after it in return type declarations, according to configuration.',
+            array(
+                new VersionSpecificCodeSample(
+                    "<?php\nfunction foo() : string {};",
+                    $versionSpecification
```

Comment:
That should probably be a fix against `master`, or do you have another idea? 

That is, following #2354, if we end up with no code examples satisfied by the current version, we display a note about that?